pylint/pyreverse/dot_printer.py

```diff
@@ -10,13 +10,19 @@
 import subprocess
 import sys
 import tempfile
+from enum import Enum
 from pathlib import Path
 
 from astroid import nodes
 
 from pylint.pyreverse.printer import EdgeType, Layout, NodeProperties, NodeType, Printer
 from pylint.pyreverse.utils import get_annotation_label
 
+
+class HTMLLabels(Enum):
+    LINEBREAK_LEFT = '<br ALIGN="LEFT"/>'
```

Comment:
It's only used in the dot printer so let's keep it in the dot printer as a nested class :)

Reply:
@Pierre-Sassoulas do I have to change the `HTMLLabels` class?

Reply:
You can make a single variable instead of an enum too, but both work.

Reply:
@Pierre-Sassoulas  I think it's better to have it as a class in case more labels are added in the future